src/MapLayers/Jobs.js

```diff
@@ -243,9 +243,23 @@ function Jobs(props) {
       planes: props.options.planes[marker],
       siminfo: s.display.sim,
       actions: props.actions,
-      id: 'jobs'+color
+      id: 'jobs'+color,
+      allJobs: group.getLayers().filter(x => x.options.fromIcao === marker || x.options.toIcao === marker)
```

Comment:
This is quite compute intensive, it increase a lot the render time (x3 in my quick test)